protocol-dashboard/src/types.ts

```diff
@@ -168,6 +168,7 @@ export type ProposalEvent = {
 }
 
 export enum Status {
+  Idle = 'Idle',
```

Comment:
What's this for? A lot of components check for Loading so we might have to add check for Idle as well.

Reply:
I didn’t mean to include Idle in the commit actually so ill remove in another PR. It was originally to match the pattern we have on web client, but that's a bigger ask. it's unused in this PR atm

Reply:
ohh ok thanks sorry for not catching this before i approved lol